src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReportingService.kt

```diff
@@ -27,12 +28,12 @@ class ReportingService(
     asyncJobLauncher.run(
       performanceReportJob,
       JobParametersBuilder()
-        .addString("user.id", user.id)
         .addString("user.firstName", userDetail.firstName)
         .addString("user.email", userDetail.email)
         .addDate("from", batchUtils.parseLocalDateToDate(from))
         .addDate("to", batchUtils.parseLocalDateToDate(to.plusDays(1))) // 'to' is inclusive
         .addString("timestamp", Instant.now().toEpochMilli().toString())
+        .addParameter("authUser", AuthUserJobParameter(user))
```

Comment:
_nit_ There's

```
    // this is really not ideal - but the intention here is validate the user has
    // a valid access scope before launching the job (which will also call this method).
    serviceProviderAccessScopeMapper.fromUser(user)
```
above

What if we renamed it to `serviceProviderAccessScopeMapper.validatedScopeFrom(user)`? It'd make the comment unnecessary, as it'd be obvious from the code that it validates access

Reply:
Yeah, I agree that the `fromUser` method doesn't really explain what is happening. I think renaming the method to `obtainAccessScope(user)` and the class `ServiceProviderAccessAuthorizator`?